tests/function-controller/pkg/servicebindingusage/servicebindingusage.go

```diff
@@ -39,6 +39,10 @@ func New(name, usageKind string, c shared.Container) *ServiceBindingUsage {
 	}
 }
 
+func (s ServiceBindingUsage) GetName() string {
```

Comment:
Just like in the previous comment - for the sake of consistency, I'd use pointer receiver here and change the receiver name to `sbu` (and in all the other `GetName` methods).